first draft of funding requests (#53)
astropy/project: finance/funding_requests.md

> Funding requests can be made to the Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 

Reviewer: Interim

> Funding requests can be made to the Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 

Reviewer 2: Suggest adding a preamble like:

The Astropy project wants to make life easy for all the volunteers and people working towards our shared goal; on the other hand, we also need to make sure we spend grant money in a fiscally responsible way and as allowed by donors or funding agencies. This policy thus aims to strike a balance between as little bureaucracy as possible while also ensuring that money is spend responsibly and that Astropy has all the documentation that we required for any of the grants given to Astropy.

If you have suggestions to improve these policies, please open an issue or pull request in https://github.com/astropy/project/ . If you have questions on how to apply the rules, please email inance@astropy.org .

> ## Travel

Reviewer 3: I don't think it need be covered in the scope of this PR (which, per our meeting earlier, is basically about formalizing  existing arrangements), but: what about advances?

> 1. Send an email to astropy-finance-committee@googlegroups.com with the following information:

Reviewer 4: ```suggestion
1. Send an email to finance@astropy.org with the following information:
```

> 1. Send an email to astropy-finance-committee@googlegroups.com with the following information:

Reviewer 2: finance@astropy.org

> 1. Send an email to astropy-finance-committee@googlegroups.com with the following information:

Reviewer 3: @kelle 's comment below alludes to this, but... how far in advance? Does it matter? Presumably we want to encourage submission in good time, both so we have plenty of time to discuss it and so that airfares etc don't climb too high.

* Name
* Description
* Total Request
* Itemized Budget

> Costs for per diem can use the [US Government per deim rates | https://www.gsa.gov/travel/plan-book/per-diem-rates], and airfare costs should be justified. 

Reviewer 4: "should be justified" is a bit ambiguous... What counts as justification?  Although perhaps this really requires a clear airfare policy (wich is pending) to really be clear?

> Costs for per diem can use the [US Government per deim rates | https://www.gsa.gov/travel/plan-book/per-diem-rates], and airfare costs should be justified. 

Reviewer 3: ```suggestion
Costs for per diem can use the [US Government per diem rates | https://www.gsa.gov/travel/plan-book/per-diem-rates], and airfare costs should be justified. 
```

Not clear to me from the above — do we expect people to claim the federal per diem, or do they have the option of submitting itemized receipts? In fact, are we just using the per diem rates to estimate expenses, and will still require receipts before reimbursement?

Reviewer 3: Also — do we use the State Department rates for non-US travel? They are often _very_ high.

> 2. The financd committee will review the costs and provide approval within 1 week of the request. 

Reviewer: interim finance committee

Reviewer: will aim to provide approval within 1 week of the request.

Reviewer: Ideally, at least 1 month before travel is to commence. Include relevance to the Project-wide goals.

Reviewer: Refer to the travel guidelines issue....which is still in the private repo. It was on my list to move them...boo....

> 2. The financd committee will review the costs and provide approval within 1 week of the request. 

Reviewer 4: ```suggestion
2. The finance committee will review the costs and provide approval within 1 week of the request. 
```

3.  Travel occurs

4.  Preferable within two weeks of completing the trip, travel costs are submitted via the [NumFOCUS Rocket form | https://numfocus.org/rocket]

5. NumFOCUS provides the information for approval to the Finance Committee.

6. At least, one member of the finance committee provides approval for the payment.

> 7. Payment is made to the requester for travel. 

Reviewer: payment is made by NumFocus, typically within 30 days.

## Contractor Work

> 

Reviewer 2: This document describes how the process has been handled in the past, when funding was available through specific grants for very well defined tasks. The interim finance committee and the community as a whole are engaging in a discussion how to handle this going forward.

> 1.  A request for contract work should be made via email to astropy-finance-committee@googlegroups.com.  

Reviewer: Requests should be for a substantive amount of work (e.g., at least 30 hours/month). The request should include a description of the proposed work, relevance to project-wide goals, and suggested method of payment (e.g., subaward to institution, hourly contract).

Reviewer 2: I generally agree with @kelle but that's exactly the point I wanted to get to in the end: How do we decide who gets paid for exactly how much work? In practice, I imagine that we might select which issues need to be worked on and have an application process or some way to work out whom of all the qualified project members to assign to that.  
But what I'm saying here is the process to be worked out - we don't have that yet and we should say so.

> 1.  A request for contract work should be made via email to astropy-finance-committee@googlegroups.com.  

Reviewer 3: ```suggestion
1.  A request for contract work should be made via email to finance@astropy.org.  
```

> 2.  If the request is not coming from the coordination committee, the finance committee will seek approval for this contract.   

Reviewer: will seek approval from the CoCo. (This sentence is just not clear.)

3.  The terms of the contract work are negotiated (by FC? by NumFOCUS?)  

> 4.  The contract is approved by 60% (?) of the finance committee.

Reviewer: (3/5)

Reviewer: majority (>50%)

Reviewer: In cases where a substantial portion of the committee is conflicted on the decision, approval requires unanimous approval of the unconflicted members.

> 4.  The contract is approved by 60% (?) of the finance committee.

Reviewer 4: ```suggestion
4.  The contract is approved by majority of the finance committee.
```

5.  The contract begins.  Hours worked are reported through the [NumFOCUS Rocket form | https://numfocus.org/rocket]. 

6.  NumFOCUS provides the information for approval to the Finance Committee.

7.   At least one member of the finance committee provides approval for the payment.  

8.   Payment is made to the contractor.

> 9.   Contracts should be reviewed yearly.   If necessary, renewals or extensions can be requested via email to astropy-finance-committee@googlegroups.com.

Reviewer: finance@astropy.org

## Any other requests for funding

> Any other requests for funding should be made via an email to astropy-finance-committee@googlegroups.com.   Please include your name, a description of the funding to be requested, and a budget. 

Reviewer 3: ```suggestion
Any other requests for funding should be made via an email to finance@astropy.org.   Please include your name, an explanation of the funding being requested, and a budget. 
```

> 

Reviewer: The interim finance committee may need to revise this policy on an a case-by-case basis as needed and will make every effort to revise this policy as needed.

The Astropy project wants to make life easy for all the volunteers and people working towards our shared goal; on the other hand, we also need to make sure we spend grant money in a fiscally responsible way and as allowed by donors or funding agencies. This policy thus aims to strike a balance between as little bureaucracy as possible while also ensuring that money is spend responsibly and that Astropy has all the documentation that we required for any of the grants given to Astropy.

> 

Reviewer: Add something here about how the anticipated costs for the first couple years will be for folks already heavily involved in the project.

Funding requests can be made to the Interim Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
If you have questions on how to apply the rules, please email finance@astropy.org .

## Travel

Travel requests should be made as early as possible and at least 1 month before travel.   Requests made less than 1 month before the travel is to occur will still be considered but should include a justification for the timing.  

1. Send an email to finance@astropy.org with the following information:
* Name
* Description
* Total Request
* Itemized Budget
* Advance needed

> Costs for per diem can use the [US Government per diem rates | https://www.gsa.gov/travel/plan-book/per-diem-rates].  These should be considered maximum values for the trips and request less than that amount can be submitted for reimbursement.

Reviewer 4: ```suggestion
If using per diem, the requestor can use the [US Government per diem rates | https://www.gsa.gov/travel/plan-book/per-diem-rates].  These should be considered maximum values for the trips and requests less than that amount can be submitted for reimbursement.
```

> Flights should be booked in economy class.   If the flight is over 4 hours

Reviewer 4: "should" might be interpreted as "must" by some readers, but this suggestion I think clears that up a bit
```suggestion
In general, flights should be booked in economy class.   If the flight is over 4 hours
```

and the additional cost is less than 20%, a higher class of seating may be
booked.  We encourage travel plans to be optimize their travel for
productivity.   If additional costs are required, a justification should be 
submitted for approval.  Medical reasons are an acceptable justification 
for higher costs.   

In general, contributors should attempt to minimize the environmental impact

> of their travel. 

Reviewer 4: Do we want to explicitly say "even if it costs a bit more"?  I.e. something like:
```suggestion
of their travel. This can, for example, justify moderately more expensive flights.
```

2. The interim finance committee will aim to provide approval within 1 week of the request.

3.  Travel occurs.

4.  Preferable within two weeks of completing the trip, travel costs are submitted via the [NumFOCUS Rocket form | https://numfocus.org/rocket]

5. NumFOCUS provides the information for approval to the Finance Committee.

> 6. At least, one member of the finance committee provides approval for the payment.

Reviewer 4: ```suggestion
6. At least one member of the finance committee provides approval for the payment.
```

7. Payment is made by NumFocus, typically within 30 days.

## Contractor Work

> This document describes how the process has been handled in the past, when funding was available through specific grants for very well defined tasks. The interim finance committee and the community as a whole are engaging in a discussion how to handle this going forward.

Reviewer 3: The above works for me!

> 

Reviewer: maybe repeat at the beginning as well?

1.  A request for contract work should be made via email to finance@astropy.org.

> 2.  If the request is not coming from the coordination committee, the finance committee will seek approval for this contract from the Coordination Committee   

Reviewer 4: Allowing for the possibility like we've done thus far of the CoCo saying "this is some contract work that's desired" and just not requiring a second "is that ok?"
```suggestion
2.  If the request is not coming from or already solicited by the coordination committee, the finance committee will seek approval for this contract from the Coordination Committee   
```

> 3.  The terms of the contract work are negotiated.

Reviewer 4: Do we want to add a step before this one along the lines of "negotiate a scope of work"?  That's sort of a sub-point of "terms of contract work" but in practice that has been a distinct step for most cases.

> 4.  The contract is approved by a majority (>50%) of the finance committee.  In cases where a substantial portion of the committee is conflicted on the decision, approval requires unanimous approval of the unconflicted members.

Reviewer 4: ```suggestion
4.  The contract is approved by a majority (>50%) of the finance committee.  In cases where a substantial portion of the committee has a conflict of interest on the decision, approval requires unanimous approval of the unconflicted members.
```
(ambiguous whether "conflict" in the original wording meant "disagree" vs "Conflict of Interest")

5.  The contract begins.  Hours worked are reported through the [NumFOCUS Rocket form | https://numfocus.org/rocket]. 

6.  NumFOCUS provides the information for approval to the Finance Committee.

7.   At least one member of the finance committee provides approval for the payment.  

8.   Payment is made to the contractor.

> 9.   Contracts should be reviewed yearly.   If necessary, renewals or extensions can be requested via email to finance@astropy.org.

Reviewer 4: ```suggestion
9.   Contracts should be reviewed at least yearly.   If necessary, renewals or extensions can be requested via email to finance@astropy.org.
```
allowing for the point Matt made that he *wanted* a more frequent check-in.
